### ISSUE TITLE
Parser error

### ISSUE THREAD

**Issue by sandlerr** [seattlerb/ruby_parser#249]
Hi there,
I am seeing an error in brakeman parsing some Ruby code that looks ambiguous but works in a specific way:

```ruby
mount (Class.new do
  def initialize
  end
end).new, at: 'endpoint'
```

this can be worked around by editing the source code to be more readable:

```ruby
mount((Class.new do
  def initialize
  end
end).new, at: 'endpoint')
```

I suspect the parser is expecting the closing bracket before the start of the block, but in this case the brace was intended to make the block be passed to the `Class` constructor rather than to the `mount` method.

**Comment by presidentbeef** [comment]
Possibly related to https://github.com/seattlerb/ruby_parser/commit/adeceebf04f03bd0a878201fbf5979111acaa26f based on the commit message.

**Comment by zenspider** [comment]
Sorta... But around command calls, not classes/methods.

This should be fixed (2.0+)
